src/coreclr/src/.nuget/packaging.props

```diff
@@ -25,6 +25,14 @@
     <VersionTxtFile Condition="'$(VersionTxtFile)' == ''">$(ArtifactsObjDir)version.txt</VersionTxtFile>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Set the boolean below to true to generate packages with stabilized versions -->
+    <IsShipping>false</IsShipping>
```

Comment:
In libraries this property is set to true for anything except tests. Why is this different in coreclr? Based on the [official docs](https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#isshipping-isshippingassembly-isshippingpackage-isshippingvsix-bool), `IsShipping` determines if an asset is...

>  if it is intended to be delivered to customers via an official channel

Why do we set this to false for coreclr packages then, which are shipping to customers?

Reply:
Because we're marking them as non shipping for servicing releases since we don't want to ship them (publish to nuget). The difference with libraries, is that the libraries packages are selectively built, this are always built. I'm just moving this from one place to another to keep the fix simple.

Reply:
That makes sense, thanks for clarifying. Do you know if we have an issue regarding building and publishing packages across the repo the same way?

Reply:
I don't think so... @Anipik do you know?

Reply:
no we donot

Reply:
@Anipik Do you think it would be worth improving the current situation where packaging and publishing properties differ? If yes, could old file one?